src/neo4clj/client.clj

```diff
@@ -128,33 +164,32 @@
   :nodes - collection of neo4j node representations
   :rels  - collection of neo4j relationship representations
   :returns - collection of aliases to return from query"
-  [conn ^clojure.lang.APersistentMap graph]
-  (execute! conn (builder/create-graph-query graph)))
+  [runner ^clojure.lang.APersistentMap graph]
+  (execute! runner (builder/create-graph-query graph)))
 
 (defn get-graph
   "Lookups the nodes based on given relationships and returns specified entities
 
   Format of the graph is:
   :nodes - collection of neo4j node representations
   :rels  - collection of neo4j relationship representations
-  :returns - collection of aliases to return from query
-  :unique-by - reference-id to use for creating a list of distinct nodes"
-  [conn ^clojure.lang.APersistentMap graph]
-  (execute! conn (builder/get-graph-query graph)))
+  :returns - collection of aliases to return from query"
+  [runner ^clojure.lang.APersistentMap graph]
+  (execute! runner (builder/get-graph-query graph)))
 
 (defn add-labels!
   "Takes a collection of labels and adds them to the found neo4j nodes"
-  [conn
+  [runner
    ^clojure.lang.APersistentMap neo4j-node
    ^clojure.lang.APersistentVector labels]
-  (execute! conn (builder/modify-labels-query "SET" neo4j-node labels)))
+  (execute! runner (builder/modify-labels-query "SET" neo4j-node labels)))
```

Comment:
These are good examples of the responsibility of query builder leaking into the client. It makes a good case for doing something similar with `"RETURN ..."` in some cases. This should not be changed in this PR... just seen as an observation.